VideoLocker/src/main/java/org/edx/mobile/view/CourseDetailInfoActivity.java

```diff
@@ -31,6 +31,8 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
+        setTitle(R.string.announcement_title);
+
```

Comment:
This should be changed in the `label` attribute in the manifest entry for this `Activity`, instead of dynamically setting it.